cves/CVE-2006-3747.yml

```diff
@@ -203,4 +223,7 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: It seems like the fault was in the code. The fix was a simple change
```

Comment:
Could you potentially expand upon this section? You get right to the answer but I feel like you could provide more detail.